ports/cpuid/vcpkg.json

```
@@ -4,7 +4,7 @@
   "description": "Provides CPU identification for the x86 (and x86_64)",
   "homepage": "https://github.com/anrieff/libcpuid",
   "license": "MIT",
-  "supports": "(x86 | x64 | arm | linux) & !uwp",
+  "supports": "(x86 | x64 | arm) & linux & !uwp",
```

IIUC it builds and install on all x86/x64/arm platforms. Only the capabilties vary.
uwp might be a different story.
```suggestion
  "supports": "(arm | x64 | x86) & !uwp",
```